frontend/kesaseteli/employer/src/hooks/wizard/useStepStorage.ts

```diff
@@ -1,16 +1,14 @@
 import useApplicationApi from 'kesaseteli/employer/hooks/application/useApplicationApi';
-import { getStepNumber } from 'kesaseteli/employer/utils/application-wizard.utils';
 import useLocalStorage from 'react-use/lib/useLocalStorage';
 import LocalStorageApi from 'shared/types/localstorage-api';
 
 const useStepStorage = (
-  stepType: 'current' | 'last-visited'
-): LocalStorageApi<number> => {
+  stepType: 'current' | 'last-completed'
```

Comment:
should it be enums?

Reply:
Perhaps but I like it more this way, at least my IDE can autocomplete possible values when calling the method.